x-pack/plugins/apm/public/components/app/RumDashboard/RumHome.tsx

```diff
@@ -16,7 +16,7 @@ export function RumHome() {
         <EuiFlexGroup alignItems="center">
           <EuiFlexItem grow={false}>
             <EuiTitle size="l">
-              <h1>End User Experience</h1>
+              <h1>Client Side Monitoring</h1>
```

Comment:
Should this be translated?

Reply:
I am leaving this for now as is.

Reply:
I think it makes sense to translate this before we merge it

Reply:
@shahzad31 do you want to add a translation here?

Reply:
done